components/config-mgmt-service/grpcserver/cfg_mgmt.go

```diff
@@ -315,6 +312,30 @@ func (s *CfgMgmtServer) GetSourceFqdns(ctx context.Context,
 	return sourceFqdns, err
 }
 
+func removeSuggestionTypeFromFilter(filter []string, suggestionType string) []string {
+	adjustedFilter := make([]string, 0)
+	for _, filter := range filter {
+		keyValuePair := strings.Split(filter, ":")
+
+		if len(keyValuePair) == 2 && keyValuePair[0] != suggestionType {
```

Comment:
What do we do with filters that have `:` in the value? `len(keyValuePair)` will be > 2 in that case, right?

Reply:
Good point. 🤔  I will create an issue to look into problem with search terms with `:` in them.

Reply:
@alexpop This has not been a problem yet because the value from the UI is URL encoded. So, "chef:v2" would be "chef%253Av2*". This would be a problem for an API user if they do not encode the value.

Reply:
Ok, thank you